### PR TITLE
FE-2408 fixed console errors and story crashing during onChange event…

### PR DESCRIPTION
**Pull request by damianOfKyan**
# Description

changed onClick propType to PropTypes.func
changed storybook store value being passed to the component to be in line with propType.

### CODE REVIEW

File: change_log/next/FE-2408-inline-input-error.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Bug Fixes:
+  - "Fixes issue were propType of onClick was expected to be an object rather than function. (Component: InputIconToggle)"
```
Comment (grabkowski) [inline change_log/next/FE-2408-inline-input-error.yml]: a typo: `were`

Comment (damianOfKyan) [reply change_log/next/FE-2408-inline-input-error.yml]: `were` is a correct form here.

Comment (damianOfKyan) [reply change_log/next/FE-2408-inline-input-error.yml]: apologies, changed to `where`.

Comment (jamime) [reply change_log/next/FE-2408-inline-input-error.yml]: shouldn't it be `when` or `where`?

**Pull request merged**